package.json

```diff
@@ -9,7 +9,8 @@
 	"dependencies": {
 		"apify": "^3.2.6",
 		"crawlee": "^3.11.5",
-		"puppeteer": "*"
+		"puppeteer": "*",
+		"uuid": "^11.0.4"
```

Comment:
Node now has UUID generation built in: https://nodejs.org/api/crypto.html#cryptorandomuuidoptions

That way you don't need to install any extra packages, and that will speed up your builds a bit 🙂